test/cpp/end2end/client_lb_end2end_test.cc

```diff
@@ -1269,31 +1268,48 @@ TEST_F(ClientLbEnd2endTest, RoundRobinWithHealthCheckingInhibitPerChannel) {
 
 class ClientLbInterceptTrailingMetadataTest : public ClientLbEnd2endTest {
  protected:
+  static void SetUpTestCase() {
```

Comment:
Who will call `SetUpTestCase()` and `TearDownTestCase()`?

Reply:
It is the way gtest sets up some global state for all the tests. Assuming this sequence:
```
ClientLbTest::SetUpTestCase();
{
  ClientLbTest test;
  test.SetUp();
  test.Test_1();
  test.TearDown();
}
{
  ClientLbTest test;
  test.SetUp();
  test.Test_2();
  test.TearDown();
}
ClientLbTest::TearDownTestCase();
```
The new name is `SetUpTestSuite`.